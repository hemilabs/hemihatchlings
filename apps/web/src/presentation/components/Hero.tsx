import React from 'react'
import logo from '../../assets/logo.svg'

function Hero(): JSX.Element {
  return (
    <div className="flex flex-col justify-center items-center h-24">
      <img src={logo} className="" alt="logo" />
      <h2 className="font-poller text-white text-center text-xs mt-4
        sm:text-lg md:text-xl lg:text-3xl">
        Learn how Ethereum & Bitcoin Finality Works.
      </h2>
    </div>
  )
}

export default Hero
