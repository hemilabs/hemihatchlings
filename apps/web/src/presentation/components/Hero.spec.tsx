import { describe, it, expect} from 'vitest'
import React from 'react'
import { render } from '@testing-library/react'
import Hero from './Hero'

describe('src/presentation/Hero', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Hero />)

    expect(asFragment()).toMatchSnapshot()
  })
})
