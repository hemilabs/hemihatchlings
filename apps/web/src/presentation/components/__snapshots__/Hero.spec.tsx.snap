// Vitest Snapshot v1

exports[`src/presentation/Hero > renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col justify-center items-center h-36"
  >
    <img
      alt="logo"
      class=""
      src="/src/assets/logo.svg"
    />
    <h2
      class="font-poller text-white text-3xl mt-8"
    >
      Learn how Ethereum & Bitcoin Finality Works.
    </h2>
  </div>
</DocumentFragment>
`;
